package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "codesplain-lambdas",
+  "version": "0.1.0",
```

Comment:
The versioning is done on the tag/S3 path level, so it's not necessary in the lambdas, so I don't think it matters how we handle it in the package.json.